ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -43,6 +43,7 @@ class ListDemoController: DemoController {
         listCell.onTapAction = {
             self.showAlertForAvatarTapped(name: samplePersonas[4].name)
         }
+        listCell.hasDivider = true
```

Comment:
Are we changing the default behavior of listCell to not have dividers unless explicitly set to true?

Reply:
Yep!

Reply:
are you documenting that behavior somewhere?

Reply:
Looks like it got missed on the description, will add to iteration!